openedx/core/djangoapps/kafka_consumer/consumers.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from django.conf import settings
+
+from confluent_kafka import KafkaError, KafkaException, DeserializingConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.schema_registry.avro import AvroDeserializer
+from confluent_kafka.serialization import StringDeserializer
+
+logger = logging.getLogger(__name__)
+
+
+# TODO EVENT BUS:
+# Use TrackingEvent class from openedx_events and use Attr <-> Abro bridge to deserialize
+class TrackingEvent:
+    """
+    Copy of TrackingEvent from license-manager to represent license manager events
```

Comment:
Adding comment about the temporary nature of this.
```suggestion
    Copy of TrackingEvent from license-manager to represent license manager events
    
    Note: this definition is temporary, and would be replaced by a definition in openedx-events.
```

Also, because this is so temporary, I am giving it very little attention in review.